### ISSUE TITLE
[Question] access to simulation parameters via *.json

### ISSUE THREAD

**Issue by PavelBlend** [InteractiveComputerGraphics/SPlisHSPlasH#25]
Can DynamicBoundaryDemo.exe and StaticBoundaryDemo.exe use the json scenes to get these simulations?:
https://youtu.be/D_nEhix1G-w

Does json have access to all simulation parameters?
If so, where can I find the full list of parameters?

**Comment by janbender** [comment]
Yes. The simulations were actually performed using SPlisHSPlasH. The json parameters are generated by generic parameters, so you have access to all relevant parameters. However, a complete list of the parameters is still missing. But we plan to write some documentation about the parameters soon. 

You can take a look at the scene CoilingModel_Weiler2018.json which uses the same parameters.

**Comment by PavelBlend** [comment]
How to make the emitter move?

**Comment by janbender** [comment]
This was hard-coded. There is no scene parameter for that yet.

**Comment by PavelBlend** [comment]
Are you planning to add access to all simulation parameters via json in the future (including transformation collider and emitters)? And the ability to animate any parameters of the simulation (viscosity, density, etc.).

**Comment by janbender** [comment]
All simulation parameters are accessible via json. The simulation methods use our GenericParameters. Each time a parameter is added it is automatically also added to the json scene importer. 

The animation of parameters would be a nice feature but requires some work. 

I just started to write a bit documentation about the scene files and a list of the parameters. It will be soon online.
